apps/dashboard/test/system/files_test.rb

```diff
@@ -255,9 +261,8 @@ class FilesTest < ApplicationSystemTestCase
 
       within_window edit_window do
         find('#editor').click
-        find('textarea.ace_text-input', visible: false).send_keys 'foobar'
+        find('textarea.ace_text-input', visible: false).send_keys('foobar')
 
-        find('.navbar-toggler').click
```

Comment:
apparently older tests had the navbar collapsed. This is no longer needed - the navbar is now displayed by default.